frontend/src/index.tsx

```diff
@@ -21,6 +21,9 @@ const cache = new InMemoryCache({
     SampleMetadata: {
       keyFields: ["primaryId"],
     },
+    Tempo: {
+      keyFields: ["smileTempoId"],
+    },
```

Comment:
Configured GraphQL on the frontend to cache Tempo node data based on their `smileTempoId` (required for optimistic response).